src/ui/LightDarkImg/LightDarkImg.test.tsx

```diff
@@ -0,0 +1,68 @@
+import { render, screen } from '@testing-library/react'
+import React from 'react'
+import { describe, expect, it, vi } from 'vitest'
+
+import { Theme, ThemeContext } from 'shared/ThemeContext/ThemeContext'
+
+import LightDarkImg from './LightDarkImg'
+
+const renderWithTheme = (ui: React.ReactElement, theme: Theme) => {
+  return render(
+    <ThemeContext.Provider value={{ theme, setTheme: vi.fn() }}>
+      {ui}
+    </ThemeContext.Provider>
+  )
+}
```

Comment:
Thoughts on converting this to a `wrapper` component, similar to how we do it in other tests?